pkg/cmd/cli/cmd/version.go

```diff
@@ -128,38 +128,37 @@ func (o VersionOptions) RunVersion() error {
 			return
 		}
 
-		ocVersionBody, err := oClient.Get().AbsPath("/version/openshift").Do().Raw()
```

Comment:
won't the `oClient, kClient, err := o.Clients()` call above fail on a non-openshift server?